skopt/callbacks.py

```diff
@@ -0,0 +1,93 @@
+from collections import Callable
+from time import time
+
+import numpy as np
+
+
+def check_callback(callback):
+    """
+    Check if callback is a callable or a list of callables.
+    """
+    if callback is not None:
+        if isinstance(callback, Callable):
+            return [callback]
+        elif (isinstance(callback, list) and
+              all([isinstance(c, Callable) for c in callback])):
+            return callback
+        else:
+            raise ValueError("callback should be either a callable or "
+                             "a list of callables.")
+    else:
+        return []
+
+
+class VerboseCallback(object):
+    """
+    Callback to control the verbosity.
+
+    Parameters
+    ----------
+    * `n_init` [int, optional]:
+        Number of points provided by the user which are yet to be
+        evaluated. That is where `len(x0)` is `y0` is None.
```

Comment:
"That is where `len(x0)` is `y0` is None." wat? I think there is a word missing or something?